### DISCUSSION TITLE
@craco/craco vs react-app-rewired

### DISCUSSION THREAD

**Discussion by PrettyCoffee** [dilanx/craco#491] [Q&A] [answered]
Hey, just wanted to ask what the up- and downsides are compared to `react-app-rewired`.

What I could find in my research:
- both have the same functionality in a different config file
- craco has a better documentation
- both have different plugins
- meta numbers seem to be pretty equal (https://npmcompare.com/compare/@craco/craco,react-app-rewired)

But if you only compare the main packages, is there a noticeable difference?

**Comment by dilanx** [comment, marked as answer]
`craco` was originally made to support newer versions of CRA officially. `react-app-rewired` seems to have been only lightly maintained by the community since CRA 2. I'm not too knowledgeable on where the state of `react-app-rewired` lies today, so someone else might be able to answer that, but `craco` is still built to support the latest version of CRA. I think the primary functionality might remain the same, but I do plan on adding any good suggested functionality to `craco`. Although, I'm not sure for how long CRA will survive with other newer React frameworks stabilizing.
